webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/ExoSpacesHamburgerNavigation.vue

```diff
@@ -3,18 +3,16 @@
     px-0
     pt-0
     class="border-box-sizing">
-    <v-row class="mx-0 clickable spacesNavigationTitle">
+    <v-row class="mx-0 spacesNavigationTitle">
       <v-list-item
-        link
-        @mouseover="openDrawer()"
-        @click="openDrawer()">
+        link>
         <v-list-item-icon class="mb-2 mt-3 me-6 titleIcon">
           <i class="uiIcon uiIconToolbarNavItem spacesIcon"></i>
         </v-list-item-icon>
         <v-list-item-content class="subtitle-1 titleLabel">
           {{ $t('menu.spaces.lastVisitedSpaces') }}
         </v-list-item-content>
-        <v-list-item-action class="my-0">
+        <v-list-item-action class="my-0" @click="toggleOpenDrawer()">
```

Comment:
This action should go to `<v-list-item>` to make the whole item clickable instead of icon only. @SaraBoutej @AzmiTouil